README.md

```diff
@@ -15,7 +15,7 @@ This Deis component is currently in beta status, and we welcome your input! If y
 
 # About
 
-The slugrunner downloads a [slug](https://devcenter.heroku.com/articles/slug-compiler) from a specified [S3 API compatible server][s3-api] and runs it based on its [Procfile](https://devcenter.heroku.com/articles/procfile).
+The slugrunner downloads a [slug](https://devcenter.heroku.com/articles/slug-compiler) from a specified minio or S3 or GCS or Azure storage service and runs it based on its [Procfile](https://devcenter.heroku.com/articles/procfile).
```

Comment:
Can these be `Minio, S3, GCS, or Azure storage service`. Also, can you provide links to each service? Also, IMO each storage service name should link to its respective website. For example: 

```
[Minio](https://minio.io/), [S3](https://aws.amazon.com/s3/), ...
```